src/projects/TodoList/components/Todo/index.js

```diff
@@ -3,10 +3,52 @@ import './style.css';
 
 class Todo extends React.Component {
   render() {
-    let className = this.props.completed ? 'completed' : 'notComplete';
+    const taskCompleteImg = require('./images/checked-box.png')
+    const taskNotCompleteImg = require('./images/unchecked-box.png')
+    const editItemImg = require('./images/edit.png')
+    const binDelete = require('./images/bin-delete.png')
```

Comment:
Couple things about this:

1) You're bringing in some images into the component, these 'require' statements don't need to be in render they can sit at the top of the file with the other 'import' statements (see below). If they sit in render then it means you're calling the file system for the image every time render runs when we don't need to do that; we only need them once.

2) The `require` keyword is the old style javascript word for importing modules into your code. Both of the following statements are the same.

```js
const React = require('react'); // Classic JS (ES5)
import React from 'react'; // New JS (ES6)
```
Can you re-write these `require` statements to `import` statements and move them to the top of the file with the other imports

Reply:
done!